packages/amplication-server/src/core/assistant/assistantFunctions.service.ts

```diff
@@ -107,6 +205,38 @@ export class AssistantFunctionsService {
     }
   }
 
+  async validatePermissions(
+    functionName: EnumAssistantFunctions,
+    context: AssistantContext,
+    args: any
+  ): Promise<boolean> {
+    const permissions = FUNCTION_PERMISSIONS[functionName];
+    if (!permissions) {
+      return false;
+    }
+
+    if (
+      permissions.paramType === AuthorizableOriginParameter.WorkspaceId &&
+      permissions.paramPath === "context.workspaceId"
+    ) {
+      return true;
```

Comment:
Wouldn't this allow to create a project in another workspace?

Reply:
no, because the workspace ID is taken from the context, this is just a way to declare that no permissions are checked (because all enum members require a definition) - see the spec file for a relevant test